inference-engine/tests/functional/plugin/cpu/shared_tests_instances/low_precision_transformations/concat_transformation.cpp

```diff
@@ -25,7 +25,7 @@ const std::vector<LayerTransformation::Params> trasformationParamValues = {
 INSTANTIATE_TEST_CASE_P(LPT, ConcatTransformation,
     ::testing::Combine(
         ::testing::ValuesIn(netPrecisions),
-        ::testing::Values(InferenceEngine::SizeVector({ 1, 1024, 16, 16 })),
+        ::testing::Values(InferenceEngine::SizeVector({ 1, 3, 16, 16 })),
```

Comment:
Just want to understand why we have this change here?

Reply:
The simplest way to make small independent improvement: test duration time was decreased.